floodrelief/settings.py

```diff
@@ -185,3 +189,9 @@ def get_list(text):
 )
 
 ADMIN_SITE_HEADER = "Keralarescue Dashboard"
+
+CORS_ORIGIN_WHITELIST = (
```

Comment:
Please move this to .env

Reply:
I dont think @abdullatheef  its necessary. It again become more difficult for beginner contributors to contribute . Also django-environ doesn't recommend adding all the setting confiugrations to .env file. It's meant to be for specific things like databases,caches,secret key,debug